website/docs/en/config/output.mdx

```diff
@@ -12,7 +12,7 @@ The top-level output key contains a set of options instructing Rspack on how and
 ## output.assetModuleFilename
 
 - **Type:** `string`
-- **Default:** `'[hash][ext][query]'`
+- **Default:** `'[hash][ext][query]' | ((pathData: PathData, assetInfo?: JsAssetInfo) => string)`
```

Comment:
Wrong place. Would you please move this to `Type`?

Reply:
![image](https://github.com/user-attachments/assets/0bff3c55-cf46-4be8-a17f-376e7945e788)

Reply:
done